src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CommonField.java

```diff
@@ -44,6 +49,13 @@ default boolean isCompoundFieldType() {
         return isCollectionFieldType() || isComplexFieldType();
     }
 
+    default DisplayContext displayContextType() {
+        return Optional.ofNullable(getDisplayContext())
+            .filter(dc -> !dc.equals("HIDDEN"))
+            .map(DisplayContext::valueOf)
+            .orElse(null);
```

Comment:
this method in case the value of display context is "HIDDEN" is going to return `null`. Which is logically wrong.

Reply:
I would also avoid wrapping nulls into Optional just to be able to use it's api. In the end is probably making the code more complicated to read